plugin/hierarchy.py

```diff
@@ -28,7 +33,21 @@
 import weakref
 
 
-HierarchyItem = Union[CallHierarchyItem, TypeHierarchyItem]
+CallHierarchyItemHelper = TypedDict('CallHierarchyItemHelper', {
+    'name': str,
+    'kind': SymbolKind,
+    'tags': NotRequired[List[SymbolTag]],
+    'detail': NotRequired[str],
+    'uri': DocumentUri,
+    'range': Range,
+    'selectionRange': Range,
+    'data': NotRequired[LSPAny],
+    # A custom field to store the first call location, given in CallHierarchyIncomingCall.fromRanges.
+    '_$ublime_call_range': NotRequired[Range]
+})
```

Comment:
Would it be a big lift to instead make this a wrapper element that would contain the original `CallHierarchyItem` as an `item` property and whatever extra data we need (call range in this case) as a separate property? Then we wouldn't need to duplicate `CallHierarchyItem` properties here and risk things getting out of sync with the original `CallHierarchyItem`.